sdk/program/src/pubkey.rs

```diff
@@ -63,7 +63,7 @@ impl From<u64> for PubkeyError {
     Hash,
     AbiExample,
 )]
-pub struct Pubkey([u8; 32]);
+pub struct Pubkey(pub [u8; 32]);
```

Comment:
What drove this change?

Reply:
Looks like it isn't needed. I was trying to use `.as_mut_ptr()` on pubkey and failing. I did not realise that `&mut pubkey` will implicitly cast to `*mut Pubkey`.